lib/streamlit/elements/slider.py

```diff
@@ -186,9 +187,14 @@ def _slider(
         check_callback_rules(self.dg, on_change)
         check_session_state_rules(default_value=value, key=key)
 
-        # Set value default.
         if value is None:
-            value = min_value if min_value is not None else 0
+            # Set value from session_state if exists.
```

Comment:
nit: maybe add a short comment on why it has to be set here via session state since this case is a bit different from other widgets.

Reply:
DONE!